packages/genesys-spark-components/src/components/stable/gux-dropdown/gux-dropdown.tsx

```diff
@@ -442,7 +442,7 @@ export class GuxDropdown {
     if (hasSlot(option, 'subtext')) {
       // TODO: use getOptionDefaultSlot(option)?.textContent.trim() once Stencil fix for assignedNodes test issue is in (v4.27.2)
```

Comment:
Create a jira ticket for this and add it to https://inindca.atlassian.net/browse/COMUI-3594